docker/DockerfileAPI

```diff
@@ -2,6 +2,8 @@ FROM tomcat:9
 
 ENV JAVA_OPTS="-Xmx16G -XX:PermSize=13G -XX:MaxPermSize=13G -XX:+UseCompressedOops"
 ENV PREFIX="/enrichmentapi"
+ENV TITLE="Signature Commons Data API"
```

Comment:
For consistency lets use: OPENAPI_TITLE & OPENAPI_DESCRIPTION

Reply:
done